src/lib/entities/episode/ui/specs/episode-card-shell.spec.ts

```diff
@@ -0,0 +1,29 @@
+import svelte from 'svelte-inline-compile';
+import { render } from '@testing-library/svelte';
+
+import { useLocalVars } from '$lib/shared/lib/jest-hacks';
+import type { Episode } from '$lib/shared/api';
+import { EpisodeCardShell } from '../..';
```

Comment:
Maybe, but with this I also wanted to test the public interface of the slice, because that's an important part of usage. I want these tests to reflect the usage of the component as closely as possible to the real one.